### ISSUE TITLE
Server announcement system

### ISSUE THREAD

**Issue by RobsonTigerPl** [TheCSMods/mc-better-stats#124]
### Question

Hey, how do i turn off announcement system on my server? There is no better statistic screen config in server folder config

**Comment by TheCSDev** [comment]
Hello, and sorry for the inconvenience!  

For servers, you may turn it off using the `betterstats.json` config file located in the `/config/` directory that is located in the server's directory. So basically, `<server-dir>/config/betterstats.json`. If it doesn't exist, create it.

From there, I have written a documentation article that explains in detail how to configure and what each config property does (you may find it by [clicking here](https://thecsdev.com/docs/betterstats/)).

Basically, in the JSON config file, you wanna define the `server-enableSAS` property, and you wanna set it to `false`.

So, recap:
1. Go to `<server-dir>/config/`
2. Find or create the `betterstats.json` file in there
3. Have the following property be defined and set to false in there:
```
{
	"server-enableSAS": false
}
```

I hope that helps. If not, feel free to let me know, and I'll try to explain it in more details. Sorry again for the inconvenience, and have a great day!

**Comment by TheCSDev** [comment]
Forgot to bring up the fact that `betterstats.json` is not created automatically, hence likely why you didn't see the file in the config folder initially. You can just create it manually if it's missing.

Hopefully this issue may be closed now. If you have any more questions regarding this or anything else, feel free to let me know.
